### PR TITLE
Update the README.md

### PR DESCRIPTION
**Pull request by lnash94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+4. To publish to maven local:
+
+        ./gradlew clean build publishToMavenLocal
+
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community. 
+
+You can also check for [open issues](https://github.com/ballerina-platform/module-ballerina-http/issues) that interest you. We look forward to receiving your contributions.
```
Comment (anupama-pathirage) [inline README.md:56]: Why this is poiniting to http module issues?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
-# ballerina-openapi
-Ballerina OpenApi-Tool
+# Ballerina-OpenAPI
+ [![Build master](https://github.com/ballerina-platform/ballerina-openapi/workflows/Build%20master/badge.svg?branch=master)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3ABuild)
+ [![Daily build](https://github.com/ballerina-platform/ballerina-openapi/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3A%22Daily+build%22)
+ [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/ballerina-openapi.svg)](https://github.com/ballerina-platform/ballerina-openapi/commits/master)
+ 
+The OpenAPI Specification is a specification, which creates a RESTFUL contract for APIs detailing all of its resources 
+and operations in a human and machine readable format for easy development, discovery, and integration. Ballerina
```
Comment (anupama-pathirage) [inline README.md:7]:
```suggestion
and operations in a human and machine-readable format for easy development, discovery, and integration. Ballerina
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+ OpenAPI tooling will make it easy for users to start development of a service documented in the OpenAPI contract
```
Comment (anupama-pathirage) [inline README.md:8]:
```suggestion
 OpenAPI tooling will make it easy for users to start the development of a service documented in the OpenAPI contract 
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,70 @@
...
+
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+   
+   * [OpenJDK](https://adoptopenjdk.net/)
+   
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+     
+### Building the Source
+
+Execute the commands below to build from source.
```
Comment (anupama-pathirage) [inline README.md:34]:
```suggestion
Execute the commands below to build from the source.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,70 @@
...
+
+        ./gradlew clean build -x test
+
+4. To publish to maven local:
+
+        ./gradlew clean build publishToMavenLocal
+
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
```
Comment (anupama-pathirage) [inline README.md:54]:
```suggestion
As an open-source project, Ballerina welcomes contributions from the community. 
```

File: README.md (hunk 1 above)
Comment (lnash94) [reply README.md]: It was a mistake, it shoud be `ballerina-openapi` as repo name. I will fix it

**Review by anupama-pathirage — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,2 +1,71 @@
-# ballerina-openapi
-Ballerina OpenApi-Tool
+# Ballerina-OpenAPI
+ [![Build master](https://github.com/ballerina-platform/ballerina-openapi/workflows/Build%20master/badge.svg?branch=master)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3ABuild)
+ [![Daily build](https://github.com/ballerina-platform/ballerina-openapi/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/ballerina-openapi/actions?query=workflow%3A%22Daily+build%22)
+ [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/ballerina-openapi.svg)](https://github.com/ballerina-platform/ballerina-openapi/commits/master)
+ 
+The OpenAPI Specification is a specification, which creates a RESTFUL contract for APIs detailing all of its resources 
+and operations in a human and machine-readable format for easy development, discovery, and integration. Ballerina
```
Comment (praneesha) [inline README.md:7]:
```suggestion
and operations in a both human and machine readable format for easy development, discovery, and integration. Ballerina
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,71 @@
...
+ OpenAPI tooling will make it easy for users to start the development of a service documented in the OpenAPI contract 
+  by generating the Ballerina service and client skeletons. The OpenAPI tools provide the following capabilities.
+ 
+ 1. Generate the Ballerina service or client code for a given OpenAPI definition. 
+ 2. Export the OpenAPI definition of a Ballerina service.
+ 3. Validate service implementation of a given OpenAPI Contract.
```
Comment (praneesha) [inline README.md:13]:
```suggestion
 3. Validate the service implementation of a given OpenAPI contract.
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,71 @@
...
+    
+The `openapi` command in Ballerina is used for OpenAPI to Ballerina and Ballerina to OpenAPI code generations. 
+The OpenAPI compiler plugin will allow you to validate a service implementation against an OpenAPI contract during
+ compile time. 
+This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.
+For more information on the operations supported, you can refer [OpenAPI Learn Page](https://ballerina.io/swan-lake/learn/using-the-openapi-tools/)
```
Comment (praneesha) [inline README.md:19]:
```suggestion
For more information on the supported operations, go to [Using the OpenAPI Tools](https://ballerina.io/swan-lake/learn/using-the-openapi-tools/).
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,71 @@
...
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
```
Comment (praneesha) [inline README.md:24]:
```suggestion
1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
```

This section has updates of the JDL 11 migration done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1381. Please change accordingly.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,69 @@
...
+This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.
+For more information on the supported operations,go to [Using the OpenAPI Tools](https://ballerina.io/swan-lake/learn/using
+-the-openapi-tools/)
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. OpenJDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other OpenJDK distribution) 
+
+>**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html).
```
Comment (praneesha) [inline README.md:27]: This sould be indented.

File: README.md [hunk 11]
```diff
@@ -1,2 +1,69 @@
...
+This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract.
+For more information on the supported operations,go to [Using the OpenAPI Tools](https://ballerina.io/swan-lake/learn/using
+-the-openapi-tools/)
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. OpenJDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other OpenJDK distribution)
+
+    >**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html).
```
Comment (praneesha) [inline README.md:27]: The note needs to be updated as per the latest content in

**Review by praneesha — APPROVED** [review]

**Pull request merged**